<launch>
    <arg name="odom_topic"         default="/far_vins_node/optimization_pose"/>
    <arg name="imu_topic"          default="/mavros/imu/data"/>
    <arg name="optical_flow_topic" default="/mavros/px4flow/raw/optical_flow_rad"/>
    <arg name="g_scale"            default="1"/>

    <node pkg="eskf_fusion" type="run_ekf_fusion_node" name="ekf_interface_fusion_node" output="screen">
        <param name="imu_topic"          value="$(arg imu_topic)" />
        <param name="odom_topic"         value="$(arg odom_topic)" />
        <param name="optical_flow_topic" value="$(arg optical_flow_topic)" />

        <rosparam file="$(find eskf_fusion)/config/ekf_fusion.yaml" />
    </node>

</launch>
